src/main/java/net/masterthought/cucumber/util/Status.java

```diff
@@ -13,7 +13,8 @@
     SKIPPED("#88AAFF"),
     PENDING("#FBB907"),
     UNDEFINED("#FBB957"),
-    MISSING("#FBB9A7");
+    MISSING("#FBB9A7"),
+    HIDDEN("");
```

Comment:
Sounds like a challenge:) can you put some thoughts in an issue, so we
would have some guidelines ?
On May 13, 2015 10:39 PM, "Kingsley Hendrickse" <notifications@github.com>
wrote:

> In src/main/java/net/masterthought/cucumber/util/Status.java
> <https://github.com/masterthought/cucumber-reporting/pull/101#discussion_r30267628>
> :
>
> > @@ -13,7 +13,8 @@
> >      SKIPPED("#88AAFF"),
> >      PENDING("#FBB907"),
> >      UNDEFINED("#FBB957"),
> > -    MISSING("#FBB9A7");
> > +    MISSING("#FBB9A7"),
> > +    HIDDEN("");
>
> I want to get rid of velocity as well - something modem - maybe handlebars
> … <#14d4ec9defc1dbdd_>
> Sent from my iPhone
>  On 13 May 2015, at 20:37, Bogdan Livadariu <notifications@github.com>
> wrote: In src/main/java/net/masterthought/cucumber/util/Status.java: > @@
> -13,7 +13,8 @@ > SKIPPED("#88AAFF"), > PENDING("#FBB907"), >
> UNDEFINED("#FBB957"), > - MISSING("#FBB9A7"); > + MISSING("#FBB9A7"), > +
> HIDDEN(""); This will require a major refactor :) The view should be
> handled by the velocity template and the logic by the java, so we would
> strip the java methods that wraps html code. I think I can come up with
> something for 0.0.26 release. … — Reply to this email directly or view it
> on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/masterthought/cucumber-reporting/pull/101/files#r30267628>
> .
>